discovery-infra/tests/base_test.py

```diff
@@ -46,24 +46,12 @@ def get_cluster_func(cluster_name: Optional[str] = None):
 
         for cluster in clusters:
             logging.info(f'--- TEARDOWN --- deleting created cluster {cluster.id}')
```

Comment:
can you please add a separate line log for TEARDOWN?
that way it is easy to see in logs when test ended. it separate the screen

Reply:
it is relevant not only to cluster

Reply:
yeah, i can add TEARDOWN log to the other fixtures
wdty?

Reply:
done